tests/brain/test_typing.py

```diff
@@ -15,10 +18,8 @@ def test_infer_typevar() -> None:
     assign_node = builder.extract_node(
         """
     from typing import TypeVar
-    MyType = TypeVar('My.Type')
+    TypeVar('My.Type')
     """
     )
-    call = assign_node.value
-    inferred = next(call.infer())
-    assert isinstance(inferred, nodes.ClassDef)
-    assert inferred.name == "My.Type"
+    with pytest.raises(InferenceError):
+        assign_node.inferred()
```

Comment:
Small nit, but it's no longer an "assign node" given the change to the parsed code.

Reply:
Cheers!